src/storage/helper.c

```diff
@@ -360,8 +360,9 @@ storageRepoGet(const String *type, bool write)
             strEqZ(cfgOptionStr(cfgOptRepoAzureKeyType), STORAGE_AZURE_KEY_TYPE_SHARED) ?
                 storageAzureKeyTypeShared : storageAzureKeyTypeSas,
             cfgOptionStr(cfgOptRepoAzureKey), STORAGE_AZURE_BLOCKSIZE_MIN, cfgOptionStrNull(cfgOptRepoAzureHost),
-            cfgOptionUInt(cfgOptRepoAzurePort), ioTimeoutMs(), cfgOptionBool(cfgOptRepoAzureVerifyTls),
-            cfgOptionStrNull(cfgOptRepoAzureCaFile), cfgOptionStrNull(cfgOptRepoAzureCaPath));
+            cfgOptionStrNull(cfgOptRepoAzureEndpoint), cfgOptionUInt(cfgOptRepoAzurePort), ioTimeoutMs(),
```

Comment:
I'm a little surprised that the Azure Host and Azure Endpoint are allowed to be Nullable - cfgOptionStrNull(cfgOptRepoAzureHost) and cfgOptionStrNull(cfgOptRepoAzureEndpoint) - why is that?

Reply:
Oops, it should be `cfgOptionStr(cfgOptRepoAzureEndpoint)`.